agent/api/task/task_windows.go

```diff
@@ -214,6 +216,11 @@ func (task *Task) addFSxWindowsFileServerResource(
 	return nil
 }
 
+// For Windows task run via the docker DaemonManager, we will run them as ContainerAdministrator
+func (task *Task) getTaskUser(container *container.Container) string {
+	return config.ContainerAdminUser
```

Comment:
Similarly, we need to check the code paths from where the docker config method is called. Setting user to containerAdministrator in all container configs can have unintended impact. Take example that a customer specifies a lower user to run the container but we set it to Admin. 

Correct approach would be to not pass the user for Windows. Keep it empty and then let the container take user as defined in the image.